content/en/data_security/agent.md

```diff
@@ -35,7 +35,7 @@ On Debian and Ubuntu, the `datadog-agent` package has a soft dependency on the `
 
 ## Information security
 
-The Datadog Agent submits data to Datadog over a TLS-encrypted TCP connection by default. As of version 6, the Agent can be configured to enforce TLS 1.2 when connecting to Datadog. If you require the use of strong cryptography, for example, to meet PCI requirements, you should use Agent v6 and set the `force_tls_12: true` setting in the Agent's configuration file.
+The Datadog Agent submits data to Datadog over a TLS-encrypted TCP connection by default. As of version 6, the Agent can be configured to enforce a minimum TLS version when connecting to Datadog. If you require the use of strong cryptography, for example, to meet PCI requirements, you should use Agent v6/7 and set the `min_tls_version: 'tlsv1.2'` setting in the Agent's configuration file.
```

Comment:
`min_tls_version` isn't available pre-`7.39.0`/`6.39.0` so we should make sure that the wording still says to use `force_tls_12: true` for versions before that.